Functions2/Task1.js

```diff
@@ -0,0 +1,28 @@
+// Завдання 1: Оголошення та виклик функції
+// 1.1. function declaration
+function rectangleArea(width, height){
```

Comment:
function name should represent an action, e.g. `calculateRectangleArea`, `getRectangleArea` etc